docs/source/_templates/autosummary/module.rst

```diff
@@ -0,0 +1,61 @@
+{{ fullname | escape | underline}}
```

Comment:
Where is this file from? It is auto-generated from some system? Could you add a reference/source as a comment on the top?

Reply:
The three files in the _template/autosummary/ folder actually define how classes, modules etc. render with autosummary. It is not automatically generated. I can add a readme at the root of the folder if you judge is necessary.

Reply:
If you copied it from some other repo or source, then please add a line saying so. If you wrote it yourself, then no reason to add anything.

Reply:
These three files have been modified from the default templates provided by autosummary. See https://sphinx-rtd-trial.readthedocs.io/en/latest/ext/autosummary.html, section "Customizing templates".
The original files can be found here: https://github.com/sphinx-doc/sphinx/tree/master/sphinx/ext/autosummary/templates/autosummary

I will add the reference in each file.